ILTransform/TestProject.cs

```diff
@@ -223,15 +223,21 @@ public static bool MakePublic(bool isILTest, ref string line, bool force)
             if (force)
             {
                 int charIndex = line.SkipWhiteSpace();
+                string preffix = line[..charIndex];
```

Comment:
See `GetIndent` - you'd need a variant that returns int and string or consider `AddAfterIndent`.  (Fix it to do all whitespace so that other callers can benefit too - I have seen tabs in these files)

Reply:
(AddAfterIndent does not seem right here)

Reply:
Thanks! I removed prefix following another comment, but I will take into account GetIndent for next changes